samples/ustream_forward_basic/src/main.c

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#include "az_ulib_result.h"
+#include "az_ulib_ustream_forward.h"
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define START_MEMORY_OFFSET 0
+static char ustream_forward_consumer_buf[15];
+static const char* ustream_forward_producer_buf = "Hello World!\r\n";
+
+typedef struct ustream_forward_consumer_context
+{
+    uint8_t* offset;
+} consumer_context;
+
+void push_callback(const az_span* const buffer, az_context* push_callback_context)
+{
+    // handle buffer
+    az_span_to_str(ustream_forward_consumer_buf, sizeof(ustream_forward_consumer_buf)/sizeof(char), *buffer);
```

Comment:
>ustream_forward_consumer_buf [](http://example.com/codeflow?start=19&length=28)

To better represent the reality, make this target address part of the `consumer_context`.